core/src/components/toast/toast.ionic.scss

```diff
@@ -0,0 +1,76 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "./toast.common";
+
+// Ionic Toast
+// --------------------------------------------------
+
+:host {
+  --background: #{globals.$ionic-color-neutral-1200};
+  --border-radius: #{globals.$ionic-border-radius-400};
+  --box-shadow: #{globals.$ionic-elevation-400};
+  --button-color: #{globals.$ionic-color-base-white};
+  --color: #{globals.$ionic-color-base-white};
+  --max-width: 343px;
+  --start: 8px;
+  --end: 8px;
+
+  @include globals.typography(globals.$ionic-body-md-medium);
+
+  // TODO(ROU-10853): replace this value with a layer token.
+  z-index: 1001;
+}
+
+.toast-wrapper {
+  @include globals.margin(auto);
+
+  display: block;
+  position: absolute;
+
+  opacity: 0.01;
+
+  // TODO(ROU-10853): replace this value with a layer token.
+  z-index: 10;
+}
+
+.toast-content {
+  @include globals.padding(globals.$ionic-space-300, globals.$ionic-space-400);
+}
+
+// Toast Button
+// --------------------------------------------------
+
+.toast-layout-baseline .toast-button-group-start {
+  @include globals.margin(null, null, null, globals.$ionic-space-400);
+}
+
+.toast-layout-stacked .toast-button-group-start {
+  @include globals.margin(globals.$ionic-space-400, globals.$ionic-space-400, null, null);
+}
+
+.toast-layout-baseline .toast-button-group-end {
+  @include globals.margin(null, globals.$ionic-space-400, null, null);
+}
+
+.toast-layout-stacked .toast-button-group-end {
+  @include globals.margin(null, globals.$ionic-space-400, globals.$ionic-space-400, null);
+}
+
+.toast-button {
+  @include globals.typography(globals.$ionic-body-md-link);
+  @include globals.padding(0);
+
+  // necessary for ripple to work properly
+  position: relative;
+
+  background-color: transparent;
+
+  overflow: hidden;
+}
+
+// Toast Icon
+// --------------------------------------------------
+
+.toast-icon,
+.toast-button-icon {
+  font-size: globals.$ionic-font-size-600;
```

Comment:
Should this be `scale` instead?

Reply:
https://github.com/ionic-team/ionic-framework/pull/29922/commits/d931e275743196e1602ab66f82fdaa5952f23f52